pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -300,7 +305,7 @@ stdenv.mkDerivation {
 
   inherit dynamicLinker expand-response-params;
 
-  # for substitution in utils.sh
+  # for substitution in utils.bash
```

Comment:
What is making these bash scripts? I know we have in the past assumed bash - but it seemed nice to make these in theory work on non-Bash shells.

Reply:
Virtually all our stuff is bash-only. Given it's not MS-DOS with max-3 file extensions, I figure it's best to be honest on use `.bash` not `.sh` :D. More seriously, editors highlight better with `.bash`: emacs will at least check the shebang, but many of these hook files don't have one.